src/test/resources/application-test.yml

```
@@ -15,14 +15,28 @@ spring:
     init:
       mode: never
+  main:
+    allow-bean-definition-overriding: true
+
+style-my-closet:
+  storage:
+    type: ${STORAGE_TYPE:s3}  # local | s3 (테스트를 위해서 현재는 s3: local)
+    local:
+      root-path: ${STORAGE_LOCAL_ROOT_PATH:.stylemycloset/storage/test}
+    s3:
+#      access-key: ${AWS_S3_ACCESS_KEY:access-key}
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)